### ISSUE TITLE
Using Keras and Deep Deterministic Policy Gradient to play TORCS

### ISSUE THREAD

**Issue by joyhuang9473** [number9473/tensorflow#7]
# Using Keras and Deep Deterministic Policy Gradient to play TORCS #

- Author: Ben Lau
- Origin: [https://yanpanlau.github.io/2016/10/11/Torcs-Keras.html](https://yanpanlau.github.io/2016/10/11/Torcs-Keras.html)
- Related:

> 300 lines of python code to demonstrate DDPG with Keras
